tests/core/test_broadcaster.py

```
@@ -31,18 +31,12 @@ def tearDown(self):
 
     def test_broadcast_address_in_ofac_list(self):
         network = Network.local()
-        client = AsyncClient(
-            network=Network.local(),
-        )
-        composer = Mock(spec=Composer)
-
         account_config = StandardAccountBroadcasterConfig(private_key=self.private_key_banned.to_hex())
 
         with pytest.raises(Exception):
```

Ruff (0.15.12)

[warning] Do not assert blind exception: `Exception`

(B017)